emails/my_library_nyc.twig

```diff
@@ -60,9 +60,10 @@
 
 <p>As an educator at a participating MyLibraryNYC school, there are millions of free resources for you to discover.</p>
 
-<p> Immediately <a href="https://sets.mylibrarynyc.org/users/start">explore and order teacher sets</a> for delivery to your school. <p>
-
-<p>Learn more about requesting individual items for pickup, as well as accessing databases and e-books.</p>
+<ul>
+    <li> Immediately <a href="https://sets.mylibrarynyc.org/users/start">explore and order teacher sets</a> for delivery to your school. </li>
```

Comment:
@nnat425 looks good.  There's an extra space in front of "Immediately" which might show up in the email.

Reply:
Fixing it now.